.browserslistrc

```diff
@@ -1,5 +1,5 @@
-chrome >= 32,
-ie >= 9,
-edge >= 13,
-firefox >= 27,
-safari >= 8,
+chrome >= 72,
+ie >= 11,
+edge >= 100,
```

Comment:
Nit: Sorry for going back to this later, for some reason I opened this again and a thought came to my mind whether we should keep the Chrome and Edge versions in sync? Edge 100 was released in March 2022 which seems a bit too recent (I understand that it is evergreen and doesn't show up in our event sample, but makes more sense to me to handle it the same way as Chrome since it is basically the same).

Reply:
Yeap was a bit skeptical on this one too. 

- The main reason for not matching them is based on our data sources, both the internal and the OS information, which are different for the two browsers.
- Both browsers have the same upstream but different release schedules so they are bound to mismatch.

Trying to find a benefit for pulling it back.

Reply:
From what I could find out, it seems to me that Edge X is built on Blink X (it appears to be that way [based on this table](https://en.wikipedia.org/wiki/Microsoft_Edge#New_Edge_release_history)) and the same is true for Chrome.

The benefit could be that we can include a larger pool of browsers without doing any more work – since we supporting Chrome since version 72, my understanding is that we don't need to check anything else to claim that we also support Edge 79.

Reply:
Edge and Chrome have the same OS base which is Chromium _(they have the same upstream)_. The rendering engine for Chromium is Blink.

The release schedule for Chrome and Edge even if converges at some point, can change independent of each other. 
- Edge https://learn.microsoft.com/en-us/deployedge/microsoft-edge-release-schedule#release-cadence
- Chrome has changed the release schedule from six weeks to four (1 day ago some news came up again, [link](https://developer.chrome.com/blog/early-stable/) 😆 )

Yeah I see your point, I will find the edge version corresponding, in time period, with Chrome 72 and update accordingly.